source/administration/changelog.md

```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.33.2, release day TBD**
+  - Fixing performance issues with the new sidebar that affects typing performance on busy servers. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

Comment:
My only concern with the term `fixing` is that it implies a complete resolution, whereas I think there is still substantial improvements to be made.

0/5, but I wonder if we should phrase as:
```suggestion
  - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```